### ISSUE TITLE
Tracks named '0' cause problems

### ISSUE THREAD

**Issue by EReckase** [GMOD/jbrowse#662]
Had a bed file with '0' as the name of the chromosome; jbrowse issues an error when trying to load those files.  Tracked it down to IndexNames.pm, this block:

```
    my $lc_name = lc $record->[0];
    unless( $lc_name ) {
        unless( $self->{already_warned_about_blank_name_records} ) {
            warn "WARNING: some blank name records found, skipping.\n";
            $self->{already_warned_about_blank_name_records} = 1;
        }
        return;
    }
```
If $lc_name is "0", it goes into this block.

**Comment by cmdcolin** [comment]
This also came up here too https://github.com/GMOD/jbrowse/issues/610

Thanks for finding this though! Might have even been two different places where this was happening

**Comment by cmdcolin** [comment]
I think this should be fixed now via 2ff5a4a. Feel free to reopen if occurs again.
